config/initializers/hypershield.rb

```diff
@@ -0,0 +1,16 @@
+Hypershield.schemas = {
+  hypershield: {
+    hide: %w[
+      auth_data_dump
+      encrypted
+      message
+      message_html
+      message_markdown
+      password
```

Comment:
We store passwords?!

Reply:
`password` is the field for draft articles which are sharable via URL.

The `?preview=` portion of the URL is its "password".

Reply:
Phew..thanks! TIL!

Reply:
`encrypted_password` is what we store on the user. `password` came with the default and we actually have a password field on the article table interestingly enough so probably best to keep it.

Reply:
comment sorting fail lol 
<img width="373" alt="Screen Shot 2020-03-04 at 4 42 43 PM" src="https://user-images.githubusercontent.com/1813380/75925730-32665180-5e37-11ea-95d6-b3caef49072b.png">